build_library/vm_image_util.sh

```diff
@@ -444,6 +445,25 @@ install_oem_package() {
     sudo rm -rf "${oem_tmp}"
 }
 
+# Write the OEM ACI file into the OEM partition.
+install_oem_aci() {
+    local oem_aci=$(_get_vm_opt OEM_ACI)
+
+    [ -n "${oem_aci}" ] || return 0
+
+    local save_BUILD_DIR=$BUILD_DIR
+    BUILD_DIR=$FLAGS_to
```

Comment:
Mixing state like this scares me, oem_aci_create should called as a different script.